Essentials/src/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -338,6 +342,17 @@ public void addFireworkMeta(final CommandSource sender, final boolean allowShort
         }
     }
 
+    private Color getColorFromRgbHexString(String color) {
```

Comment:
Refactor method to eliminate the variable `c`. Just return the appropriate values as soon as you determine them.